src/Queries/PersonsQuery.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { QueryRenderer, graphql } from 'react-relay';
+
+import environment from '../createRelayEnvironment';
+import PersonInfo from '../Queries/PersonInfo';
+
+export default class PersonsQuery extends React.Component {
+  render() {
+    return (
+      <View style={{ paddingVertical: 20 }}>
+        <QueryRenderer
+          environment={environment}
+          query={graphql`
+            query PersonsQuery($name: String!) {
+              persons(name: $name) {
```

Comment:
it is by `name` or by `id`?

Reply:
i use by name
![image](https://user-images.githubusercontent.com/9096756/33810336-94544fa2-ddea-11e7-82a2-414c69486b1f.png)